caikit_tgis_backend/generation.proto

```diff
@@ -4,8 +4,8 @@
 
  syntax = "proto3";
  package fmaas;
-
-
+ 
```

Comment:
Total NIT, but does this file include these empty whitespace lines in `vllm`? I really dislike having them, but if this is a straight `cp` from `vllm` without any modifications, that's probably better. It may also be good to write a `script` that does this `generation.proto` update in place.